### DISCUSSION TITLE
Critical Notifications aren't visible enough

### DISCUSSION THREAD

**Discussion by DiagnosiZucchero** [linuxmint/discussions#344] [Ideas]
When recieving a critical notification its visibility is not different enough from a notification with different urgency; depending on urgencies, different notifications should probably have different background colours or very visible accent colours. 
Even better would be, beeing able to modify these beheaviours inside of the notifications settings.

**Comment by igorsantos07** [comment]
Definitely, a "you must plug your laptop now or it will hibernate, dumbass" should be more prominent than "yet another screenshot saved, sir".
